UHResidentInformationAPI/V1/Infrastructure/Address.cs

```diff
@@ -0,0 +1,34 @@
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace UHResidentInformationAPI.V1.Infrastructure
+{
+    [Table("property")]
+    public class Address
+    {
+       [Column("prop_ref")]
+        [MaxLength(9)]
+        [Key]
+        public int propertyRef { get; set; }
+
+        [Column("house_ref")]
+        [MaxLength(9)]
+        public int houseRef { get; set; } 
+
+        [Column("address1")]
+        [MaxLength(9)]
+        public int addressLine1 { get; set; } 
+
+        [Column("address2")]
```

Comment:
There isn't a column for `address2` in the schema file for the property table.

Reply:
This will be because of the discrepencies between the Swagger doc and the database schema. I will update the Swagger doc. Perhaps `address1` in the database contains the full address.